### ISSUE TITLE
Localize the fundraising app

### ISSUE THREAD

**Issue by bmispelon** [django/djangoproject.com#377]
It would be a nice touch to have the donation form, thank you page, ... localized.

**Comment by codeadict** [comment]
I can do this one guys

**Comment by timgraham** [comment]
What does the implementation look like? As far as I know, we don't have any translation infrastructure in place for the Django website (besides docs which is independent). Do we have to setup a Transifex project and start pulling and pushing translations? \cc @claudep

**Comment by claudep** [comment]
Might be nice to dogfood Django's i18n :-)

**Comment by tobiasmcnulty** [comment]
It looks like Transifex has free open source accounts. I'll see about getting one set up.

If you would like to be added as a translator please let me know here or on IRC.

See also: #862, #864, #865

**Comment by tobiasmcnulty** [comment]
I asked this on IRC as well, but I don't think everyone is logged in there: Do we already have a Transifex account? Or is someone else squatting on the 'django' org slug? https://usercontent.irccloud-cdn.com/file/fyiGYl4F/Screenshot%202019-02-17%2019.31.32.png

**Comment by charettes** [comment]
@tobiasmcnulty I'm pretty sure this is where all `django/django` translations are handled

https://docs.djangoproject.com/en/2.1/internals/contributing/localizing/

https://www.transifex.com/django/django/

**Comment by claudep** [comment]
@tobiasmcnulty Please tell me your Transifex nick and I'll add you to the Django maintainer team there.

**Comment by tobiasmcnulty** [comment]
The Transifex PR (#866) has been merged, so this should be a bit more straightforward now.

I think the steps are:

1) Mark up the strings for translation in this app. In particular the snippets included elsewhere on the site and the form itself would be great to have translated (but it is all important).

2) Once that's merged, push up the new source strings to Transifex, translate them, and then pull down/commit the translations (this could be done as a separate PR or set of PRs).

The translation docs are in the README: https://github.com/django/djangoproject.com#translation

**Comment by carltongibson** [comment]
@pauloxnet Do you think this is something you might be able to drum up some volunteers to lead? (It seems like a nice idea right?)

**Comment by carltongibson** [comment]
Not really, those all sound good. _"A bit of advertising"_ 😄 — We can chat at DjangoCon to come up more of a plan. Thanks @pauloxnet!

**Comment by pauloxnet** [comment]
Yes, a bit of "ethical advertising" to boost internal projects.
Maybe we can make a list of interesting internal projects we think it worth to promote.
We can  promote this projects also in other "channels" (es: Django News, Django Chat, ...) ?

**Comment by marksweb** [comment]
@carltongibson @pauloxnet I'll pick this up. 👍

**Comment by bmispelon** [comment]
This was basically done in a41119e30356a4147b4fb857e83b6c51a931fac4. Once we have infrastructure to actually translate the site's strings the app will automatically be translated (that issue is tracked in #1648 )

I think this particular issue can therefore be closed.

**Comment by adamzap** [comment]
It looks like this is done!
